code/modules/mob/living/silicon/decoy/decoy.dm

```diff
@@ -31,16 +34,16 @@
 
 	update_stat()
 
-/mob/living/silicon/decoy/death(gibbed, deathmessage = "sparks up and falls silent...")
-	set waitfor = 0
```

Comment:
Are you sure you should remove this in the context, given there's a `sleep` function?